ambry-api/src/main/java/com/github/ambry/account/AccountService.java

```diff
@@ -103,4 +105,10 @@
    *          exist.
    */
   public boolean removeAccountUpdateConsumer(Consumer<Collection<Account>> accountUpdateConsumer);
+
+  /**
+   * It gets a collection of {@link Account}s' and {@link Container}s' Id pairs selected by
+   * {@link com.github.ambry.account.Container.ContainerStatus}.
+   */
+  public Set<Pair<Short,Short>> getContainersByStatus(Container.ContainerStatus containerStatus);
```

Comment:
nit: we don't have to explicitly mark it public since all methods in an interface is public.

Reply:
Thanks for the nit. Updated.